eng/configure-toolset.sh

```diff
@@ -0,0 +1,5 @@
+# We depend on a local cli for a number of our buildtool
+# commands like init-tools so for now we need to disable
+# using the globally installed dotnet
+
+useInstalledDotNetCli=false
```

Comment:
Per https://github.com/dotnet/arcade/issues/1468, I think we should use use_installed_dotnet_cli for the bash version.

Reply:
I copied this code from corefx.  Not sure we actually need these files.  I will try to remove.